analyses/01-cram-to-bam.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+## Define default variables
+kf_id_col=1   # KF patient ID column
+chr_col=3     # Chromosome
+pos_col=4     # Position
+label_col=11  # Additional label to add to plot for identification, i.e. gene
+window=10000  # Bases to plot either side of the position given
```

Comment:
A few notes here: 

* To make this more robust, I think this should be updated to consider both the start and end positions and make the windows around those. I think for variant cases only one position is relevant, but for splice events where the exons might be far apart, we'd want to make sure we're capturing a window around this interval for plotting. 
* I think we might want to create a standardized file format for these input files to make sure these columns indices are always correct. I think the input file should include: `ID`, `chr`, `start_pos`, `end_pos`, `gene_name` (anything else?). And then you can build in a check to make sure any input file that is supplied has these columns in this order.